### ISSUE TITLE
The entire page must not reload when we go to the dashboard tab

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/community-app#5925]
When accessing the dashboard for the first time, the entire page reloads before the dashboard is loaded.

https://user-images.githubusercontent.com/58783823/145662041-8d3809af-6000-4358-81aa-297883e6de83.mov

**Comment by ThomasKranitsas** [comment]
I don’t think this actually happens from my code. Instead, this already happens when you open a challenge

**Comment by SathyaJayabal** [comment]
right, let me remove the MM_Dashboard label.
